python/ray/actor.py

```diff
@@ -172,11 +176,27 @@ class ActorClassMetadata:
             each actor method.
     """
 
-    def __init__(self, modified_class, class_id, max_reconstructions, num_cpus,
-                 num_gpus, memory, object_store_memory, resources):
-        self.modified_class = modified_class
+    def __init__(self, language, modified_class_or_function_descriptor,
```

Comment:
`modified_class_or_function_descriptor` is a bit weird. 
`function_descriptor` should always be present.

Reply:
The code has been updated.